lib/cases/case_page.dart

```diff
@@ -52,6 +65,31 @@ class CasePage extends ConsumerWidget {
                     mainAxisAlignment: MainAxisAlignment.start,
                     children: [
                       Expanded(child: Text('Case: ${caseDocRef.id}')),
+                      // Dropdown button for selecting the status
+
+                      DropdownButton<STATUS>(
+                        isExpanded: true,
+                        value: STATUS.draft,
+                        items: getAllStatuses()
+                            .map<DropdownMenuItem<STATUS>>((STATUS value) {
+                          return DropdownMenuItem<STATUS>(
+                            value:
+                                value, // Use the enum value as the value for the DropdownMenuItem
+                            child: Text(value
+                                .name), // Use the extension to get the name of the status
+                          );
+                        }).toList(),
+                        onChanged: (newValue) async {
+                          try {
+                            updateDocumentWithStatus(caseDocRef, newValue!);
+
+                            print('Document updated successfully.');
+                          } catch (e) {
+                            print('Error updating document: $e');
+                          }
```

Comment:
Don't need to handle the exception just to print it because the probability of it is quite low. Exception will get printed when it is thrown anyway.